Tests/TestUtilities/JavaPropertyReader.cs

```diff
@@ -28,6 +28,7 @@ namespace TestUtilities;
 /// This class reads Java style properties from an input stream.
 /// </summary>
 /// Copied from https://github.com/Kajabity/Kajabity-Tools
+// ToDo: Remove this class in https://sonarsource.atlassian.net/browse/SCAN4NET-721
```

Comment:
practically used in for testing the PropertiesWriter via the SQPropertiesFileReader